src/auth/screens/events.screen.js

```diff
@@ -74,20 +77,21 @@ const styles = StyleSheet.create({
 });
 
 class Events extends Component {
-  componentDidMount() {
-    if (this.props.user.login) {
-      this.getUserEvents(this.props.user);
+  componentDidMount({ user } = this.props) {
```

Comment:
I don't think we should be including `props` as our an argument to `componentDidMount`. We never call this method as a function anywhere in the component so there's no need to set an argument and define it's default (seems redundant).

Ideally, we can just destructure within the method itself:

```
componentDidMount() {
    const { user } = this.props;
   // ....
}
```

Reply:
Oops, that's right, I fixed it.